stubs/protobuf/@tests/stubtest_allowlist.txt

```diff
@@ -3,6 +3,8 @@
 google.protobuf\..*_pb2\..*
 
 google.protobuf.internal.containers.BaseContainer.sort
+google.protobuf.internal.enum_type_wrapper.EnumTypeWrapper.DESCRIPTOR
+google.protobuf.message.Message.DESCRIPTOR
```

Comment:
I would appreciate a short comment here.

Reply:
@Akuli - I'm new to mypy stubtests but I'm seeing the following locally (linux 5.13.0-28-generic, python 3.9.5).

Without these two allowlist entries:
```
python tests/stubtest_third_party.py protobuf
...
error: google.protobuf.internal.enum_type_wrapper.EnumTypeWrapper.DESCRIPTOR variable differs from runtime type None
Stub: at line 10
google.protobuf.descriptor.EnumDescriptor
Runtime:
None

error: google.protobuf.message.Message.DESCRIPTOR variable differs from runtime type None
Stub: at line 14
google.protobuf.descriptor.Descriptor
Runtime:
None

stubtest failed for protobuf
```

With these two allowlist entries:
```
python tests/stubtest_third_party.py protobuf
...
stubtest succeeded for protobuf
```
 My interpretation of that is: while `Message` and `Descriptor` are both defined with a null `DESCRIPTOR`, sublcasses of  `Message` and instances of `EnumTypeWrapper` require this value to be set, and since These type stubs are intended for use with protoc generated python it's more accurate to make them non-nullable.

Reply:
> while Message and Descriptor are both defined with a null DESCRIPTOR, sublcasses of Message and instances of EnumTypeWrapper require this value to be set, and since These type stubs are intended for use with protoc generated python it's more accurate to make them non-nullable.

Add a comment that says this to the allowlist :)

Reply:
Yep! Your understanding is perfect.

Just need a comment in the allowlist.txt file so that someone in the future doesn't revert your change by accident.

Reply:
Ah ok, got it, done.